app/controllers/course_wikis_controller.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+class CourseWikisController < ApplicationController
+  def update
+    course_id = params[:course_id]
+    wikis = course_wiki_params[:wikis]
+    CourseWiki.where(course_id: course_id).delete_all
```

Comment:
I don't like this. It makes `created_at` and `updated_at` mostly useless. Records should only be deleted in the case where that wiki got removed.

Reply:
Got it. I will refactor it so.